README.md

```diff
@@ -63,9 +63,16 @@ For a **manual run**, execute the following commands in different terminal windo
   ```
 - Start an `essa-rs` function executor node with ID `0`:
 
+  - Use [`WasmEdge`](https://github.com/WasmEdge/WasmEdge) as the executor:
+
   ```
   cargo run --release -p essa-function-executor -- 0
   ```
+  - Use [`wasmtime`](https://github.com/bytecodealliance/wasmtime) as the executor:
+
+  ```
+  cargo run --features wasmtime_executor --release -p essa-function-executor -- 0
```

Comment:
We should include `--no-default-features`, otherwise the `wasmedge` dependencies are still built, which can cause problems on systems where `wasmedge` is not set up:

```suggestion
  cargo run --no-default-features --features wasmtime_executor --release -p essa-function-executor -- 0
```